postgresDb/generator/reviews.js

```diff
@@ -0,0 +1,26 @@
+const csvWriter = require('csv-write-stream');
+const fs = require('fs');
+var writer = csvWriter();
+var counter = 0;
+
+const reviewsGen = () => {
+  writer.pipe(fs.createWriteStream('reviews.csv'));
+  for (var i = 1; i < 100000001; i++) {
+    let stockId = Math.floor(Math.random() * Math.floor(100)) + 1;
```

Comment:
why use 'let' here?  since you're in a loop, you can use 'const'.  the loop scope is started fresh every loop iteration.  if you use the airbnb linting rules, they will point that out - could try them out for similar ideas!